docs/topics/rn-known-issues.adoc

```diff
@@ -18,21 +18,17 @@ For a complete list of all known issues, see the list of link:https://issues.red
 
 |link:https://issues.redhat.com/browse/WINDUP-3138[WINDUP-3138]
 |CodeReady Workspace plugin
-|The plugin does not work with CodeReady Workspace 2.10.
+|The plugin works with CodeReady Workspace 2.09, but not version 2.10.
 
-|link:https://issues.redhat.com/browse/WINDUP-3019[WINDUP-3019]
-|CLI
-|Bash completion feature displays an `mta-cli not found` error.
+|link:https://issues.redhat.com/browse/WINDUP-3165[WINDUP-3165]
+|Web console on OpenShift Container Platform
+|Analysis fails when MTA custom labels are uploaded and advanced options are enabled.
```

Comment:
Teeny tiny nit pick. I think the "and" should be "or". It's unlikely that analysis fails only when both conditions are true.